llvm/include/llvm/CodeGen/MachineBasicBlock.h

```diff
@@ -693,7 +693,8 @@ class MachineBasicBlock
   /// layout was modified.  If the block previously fell through to that block,
   /// it may now need a branch. If it previously branched to another block, it
   /// may now be able to fallthrough to the current layout successor.
-  void updateTerminator(MachineBasicBlock *PreviousLayoutSuccessor);
+  void updateTerminator(MachineBasicBlock *PreviousLayoutSuccessor,
+                        SlotIndexes *Indexes = nullptr);
```

Comment:
Really ought to mention `\p SlotIndexes` in the doxygen comments, here and in TargetInstrInfo.

Reply:
Done